CuitService/CuitNumber.cs

```diff
@@ -25,7 +25,7 @@ public CuitNumber(string value)
 
             var validationResult = ValidateNumber(value);
 
-            if (validationResult != ValidationResult.Success)
+            if (validationResult != null && validationResult != ValidationResult.Success)
```

Comment:
If you fix this: https://github.com/FromDoppler/cuit-service/pull/88/files#r610896052

You do not need this change.

Reply:
Meh, you need this change because of _ValidationResult.Success_ is _null_ 🤦 

(More information in the [another conversation](https://github.com/FromDoppler/cuit-service/pull/88/files#r611075267))